src/shared/components/alert-button/AlertButton.scss

```diff
@@ -0,0 +1,16 @@
+@import 'src/styles/common';
+
+.alertButton {
+  display: inline-block;
+  position: relative;
+  top: 4px;
+  cursor: pointer;
+  padding: 2px;
+  font-size: 0;
+
+  > svg {
+    width: 18px;
+    height: 18px;
+    fill: $red;
```

Comment:
We probably want this configurable through props? Two different alert levels — "error" and "warning", with error shown in red and warning shown in amber.